src/web/styles.scss

```diff
@@ -90,7 +90,6 @@ h6,
   max-height: 90vh;
   max-width: 70vw;
   min-width: 30vw;
-  overflow-y: scroll;
```

Comment:
Maybe better to add `margin-bottom: 1.75rem` to `modal-content`  so that the modal does not stick to the bottom of the page.

Reply:
I actually have another PR to refine modals: #10550. Right now let's focus to fix the important issue (modal not scrolling in Safari) first.